hieradata/clients/azure.ci.jenkins.io.yaml

```diff
@@ -267,6 +267,13 @@ profile::jenkinscontroller::jcasc:
             cpus: 4
             memory: 8
             imagePullSecrets: dockerhub-credential
+          - name: jnlp-maven-8-aio
```

Comment:
```suggestion
          - name: jnlp-maven-all-in-one
```

Reply:
I would either put this all-in-one before other jnlp-maven-* agent definitions, or either after them (l285)

Reply:
I disagree, the 8 mean that the default path for javahome on that agent will be on jdk 8, and I will build agent for 11 and 17 with the exact same image but with a specific javahome path line 276 so the agent name must be specific also to help choosing the correct one. 
But I can change 'aio' by 'all-in-one'